straxen/corrections_services.py

```diff
@@ -114,7 +116,7 @@ def _get_correction(self, run_id, correction, global_version):
                         # on when something was processed therefore
                         # don't interpolate but forward fill.
                         df = self.interface.interpolate(df, when, how='fill')
-                    if correction in corrections_w_file:
+                    if correction in corrections_w_file or correction in arrays_corrections:
```

Comment:
Okay fine for the moment I guess. But this could potentially cause problems in the long run. I think this has to be solved on the database side idk. We should store a long the correction values some metadata which explains how the correction should be interpolated between time stamps.

Reply:
yes, let me explore the idea of the metadata, see above message